OracleWebLogic/samples/12213-domain-home-in-image/container-scripts/startManagedServer.sh

```diff
@@ -53,6 +39,22 @@ if [ -z "${PASS}" ]; then
    exit
 fi
 
+#Set Java Options
+JAVA_OPTIONS=`awk '{print $1}' ${SEC_PROPERTIES_FILE} | grep ^JAVA_OPTIONS= | cut -d "=" -f2`
+if [ -z "${JAVA_OPTIONS}" ]; then 
+   JAVA_OPTIONS="-Dweblogic.StdoutDebugEnabled=false"
+fi
+export JAVA_OPTIONS=${JAVA_OPTIONS}
+echo "Java Options: ${JAVA_OPTIONS}"
+
+#Set env to Start Derby DB
+DERBY_FLAG=`awk '{print $1}' ${SEC_PROPERTIES_FILE} | grep ^DERBY_FLAG= | cut -d "=" -f2`
```

Comment:
I must have missed something. I did not see where the newly added DERBY_FLAG is checked/used?